app/models/salt_event_handlers/minion_start.rb

```diff
@@ -0,0 +1,13 @@
+class SaltEventHandlers::MinionStart
```

Comment:
I'd rename SaltEventHandlers to simply Handlers (and also the directory). I think that the SaltEvent part is already assumed. Plus, documentation